homeassistant/components/microbees/config_flow.py

```diff
@@ -0,0 +1,86 @@
+"""Config Flow for microBees."""
+
+from __future__ import annotations
+import voluptuous as vol
+import logging
+from homeassistant import config_entries
+from homeassistant.const import (
+    CONF_PASSWORD,
+    CONF_TOKEN,
+    CONF_USERNAME,
+    CONF_CLIENT_ID,
+    CONF_CLIENT_SECRET,
+    CONF_DOMAIN,
+)
+from .const import CONFIG_ENTRY_VERSION
+import json
+import aiohttp
+import base64
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class microBeesFlowHandler(config_entries.ConfigFlow, domain=CONF_DOMAIN):
+    """Handle a microBees config flow."""
+
+    VERSION = CONFIG_ENTRY_VERSION
+
+    def __init__(self):
+        """Initialize the config flow."""
+        self.data = {}
+
+    async def async_step_user(self, user_input=None):
+        """Prompt user input. Create or edit entry."""
+        errors = {}
+
+        # Login to microBees with user data.
+        if user_input is not None:
+            self.data.update(user_input)
+            async with aiohttp.ClientSession() as session:
```

Comment:
i recommend you use `hass.helpers.aiohttp_client.async_get_clientsession()` so you dont need to create a new one